### PR TITLE
Clarification of DID creation (registration)

### PR DESCRIPTION
**Pull request by martipos**
Consolidated information where it made sense, deleted redundant information and referenced to complementary information in the specification.

Signed-off-by: martipos <176692840+martipos@users.noreply.github.com>

### CODE REVIEW

File: spec/specification.md [hunk 1]
```diff
@@ -199,62 +209,87 @@ Creating a `did:tdw` DID is done by carrying out the following steps.
...
+   in step 1, with the identified placeholder for the [[ref: SCID]] (e.g., `did:tdw:{SCID}:example.com#key-1`,
+   `did:tdw:{SCID}:example.com:dids:issuer#key-1`, etc.). For the rest of the [[ref: DIDDoc]]
+   whatever content can be added as required.
+   
+   1. Note: The placeholders (the string `{SCID}`) **MUST** be in every place in the [[ref: DIDDoc]] where
+   the SCID is to be placed.
+   
+   
+4. **Generate a preliminary DID Log Entry** (input JSON array)
+   The [[ref: DID log entriy]] is an input JSON array that when completed contains the following items:
```
Comment (swcurran) [inline spec/specification.md:228]: Typo: “entriy"

File: spec/specification.md [hunk 2]
```diff
@@ -199,62 +209,87 @@ Creating a `did:tdw` DID is done by carrying out the following steps.
...
+   The [[ref: DIDDoc]] **MUST** contain the top level `id` item which **MUST** be the DID string from
+   step 1, including the placement of the `{SICD}` placeholder for the [[ref: SCID]]. Other
+   [[ref: DIDDoc]] verifications **SHOULD** be performed.
+   
+   All other absolute self-reference's to the DID in the [[ref: DIDDoc]] must use the form defined
+   in step 1, with the identified placeholder for the [[ref: SCID]] (e.g., `did:tdw:{SCID}:example.com#key-1`,
+   `did:tdw:{SCID}:example.com:dids:issuer#key-1`, etc.). For the rest of the [[ref: DIDDoc]]
+   whatever content can be added as required.
+   
+   1. Note: The placeholders (the string `{SCID}`) **MUST** be in every place in the [[ref: DIDDoc]] where
```
Comment (swcurran) [inline spec/specification.md:223]: Singular “placeholder” in this spot.

File: spec/specification.md [hunk 3]
```diff
@@ -199,62 +209,87 @@ Creating a `did:tdw` DID is done by carrying out the following steps.
...
-   and verification processes.
+      
+3. **Create the initial [[ref: DIDDoc]] for the DID**
+   The [[ref: DIDDoc]] **MUST** contain the top level `id` item which **MUST** be the DID string from
+   step 1, including the placement of the `{SICD}` placeholder for the [[ref: SCID]]. Other
+   [[ref: DIDDoc]] verifications **SHOULD** be performed.
+   
+   All other absolute self-reference's to the DID in the [[ref: DIDDoc]] must use the form defined
+   in step 1, with the identified placeholder for the [[ref: SCID]] (e.g., `did:tdw:{SCID}:example.com#key-1`,
+   `did:tdw:{SCID}:example.com:dids:issuer#key-1`, etc.). For the rest of the [[ref: DIDDoc]]
```
Comment (swcurran) [inline spec/specification.md:220]: How about for the last line — “The [[ref: DIDDoc]] can contain any other content as deemed necessary by the [[ref: DID Controller]]."

**Review by swcurran — CHANGES_REQUESTED** [review]
Looks great. Just a couple of tweaks proposed and a typo to fix.

File: spec/specification.md (hunk 3 above)
Comment (martipos) [inline spec/specification.md:220]:
```suggestion
   `did:tdw:{SCID}:example.com:dids:issuer#key-1`, etc.). The [[ref: DIDDoc]] can contain any other content as deemed necessary by the [[ref: DID Controller]].
```

File: spec/specification.md [hunk 4]
```diff
@@ -199,62 +209,87 @@ Creating a `did:tdw` DID is done by carrying out the following steps.
...
+   whatever content can be added as required.
```
Comment (martipos) [inline spec/specification.md:221]:
```suggestion
 ```

File: spec/specification.md (hunk 2 above)
Comment (martipos) [inline spec/specification.md:223]:
```suggestion
   1. Note: The placeholder (the string `{SCID}`) **MUST** be in every place in the [[ref: DIDDoc]] where
```

File: spec/specification.md (hunk 1 above)
Comment (martipos) [inline spec/specification.md:228]:
```suggestion
   The [[ref: DID log entry]] is an input JSON array that when completed contains the following items:
```

**Review by martipos — COMMENTED** [review]
Added inputs by @swcurran 
Signed-off-by: martipos <176692840+martipos@users.noreply.github.com>

**Comment by martipos** [comment]
thank you @swcurran for you inputs ! :-)

**Review by swcurran — APPROVED** [review]

**Pull request merged**